spec/factories/payment_intents.rb

```diff
@@ -23,5 +23,9 @@
       payment_record { association(:stripe_record, :payment_intent, :not_started) }
       wca_status { 'created' }
     end
+
+    trait :pending do
+      payment_record { association(:stripe_record) }
```

Comment:
In how far is the `stripe_record` that you're associating here representing the `pending` state?

Reply:
I don't quite understand the question, but the StripeRecord factory's defaults align with a pending state so I didn't have to pass any other arguments when creating the association

Reply:
That pretty much resolves my question. The fact that creating an associated `stripe_record` implicitly marks it as `pending` makes total sense, I am debating whether or not you should add a `pending` trait to the `stripe_record` as well to make it clear/explicit.

But nonetheless that's a bit out of the scope of this PR.